mmv1/third_party/terraform/services/identityplatform/resource_identity_platform_config_test.go

```diff
@@ -82,6 +82,13 @@ resource "google_identity_platform_config" "basic" {
         }
     }
   }
+  sms_region_config {
+    allow_by_default {
+      disallowed_regions = [
+        "US",
```

Comment:
As the `disallowed_regions` is a list, can you please add more items to the list?

Reply:
Done!